networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc

```diff
@@ -63,4 +63,10 @@ The Ingress Operator manages wildcard DNS. For more information, see the followi
 
 * xref:../../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[Installing a cluster on vSphere].
 
-* xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
\ No newline at end of file
+<<<<<<< HEAD
+* xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
+=======
+* xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
+
+endif::[]
+>>>>>>> ec60cf840f (Added endif in additional resources.)
```

Comment:
```suggestion
* xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
endif::[]
```
if you have lines that look like "<<<<< HEAD" it is a sign that you have an unresolved merge conflict. You can remove lines 66-68, as well as line 72.